### PR TITLE
refactor v1-based imports to db

### PR DESCRIPTION
**Pull request by alfredodeza**
Requires https://github.com/anchore/grype-db/pull/14 to be available first, so that the dependency can be bumped.

Fixes https://github.com/anchore/grype/issues/242

### CODE REVIEW

**Comment by alfredodeza** [comment]
Tests are expected to break until the go module gets updated to point to a post-refactor grype-db

File: grype/presenter/json/presenter_test.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ func TestJsonImgsPresenter(t *testing.T) {
 	}
 
 	// TODO: add me back in when there is a JSON schema
-	// validateAgainstV1Schema(t, string(actual))
+	// validateAgainstdbSchema(t, string(actual))
```
Comment (luhring) [inline grype/presenter/json/presenter_test.go:191]: nit: I'd suggest capitalizing at least the "D" in "db" to make it more identifiable as a distinct word from "Against"

**Review by luhring — APPROVED** [review]

File: grype/presenter/json/presenter_test.go (hunk 1 above)
Comment (alfredodeza) [reply grype/presenter/json/presenter_test.go:191]: I blame my search and replace regex :D will fix

Comment (alfredodeza) [reply grype/presenter/json/presenter_test.go]: This has been fixed... with an improved and polished regular expression

File: Makefile [hunk 2]
```diff
@@ -100,11 +100,6 @@ lint: ## Run gofmt + golangci lint checks
 	$(eval MALFORMED_FILENAMES := $(shell find . | grep -e ':'))
 	@bash -c "[[ '$(MALFORMED_FILENAMES)' == '' ]] || (printf '\nfound unsupported filename characters:\n$(MALFORMED_FILENAMES)\n\n' && false)"
 
-.PHONY: validate-schema
-validate-schema:
-	# ensure the codebase is only referencing a single grype-db schema version, multiple is not allowed
-	python test/validate_schema.py
```
Comment (alfredodeza) [inline Makefile:106]: @wagoodman besides the simple search and replace, this is part that got removed. There is no need to check if we are importing multiple schemas. Wanted to make sure this was visible to you before merging

Comment (wagoodman) [reply Makefile:106]: yup yup, that's correct 👍

**Review by wagoodman — APPROVED** [review]

**Pull request merged**